/**:
  ros__parameters:
    interbotix_arm:
      kinematics_solver: lma_kinematics_plugin/LMAKinematicsPlugin
      kinematics_solver_search_resolution: 0.005
      kinematics_solver_timeout: 0.005
      position_only_ik: false
